vcpkg.json

```diff
@@ -22,5 +22,11 @@
       "name": "wil",
       "platform": "windows"
     }
+  ],
+  "overrides": [
+    {
+      "name": "zlib",
+      "version": "1.2.12#2"
```

Comment:
I noticed the latest version on vcpkg is still 1.2.12. https://github.com/microsoft/vcpkg/blob/06b5f4a769d848d1a20fa0acd556019728b56273/ports/zlib/vcpkg.json#L3

How do we build the repo locally with `1.2.12#2`?

Reply:
If you're using a current vcpkg.exe, then the overrides should resolve the problem. Sync your vcpkg repo to the current master branch and I believe it should resolve the issue.